File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long long rev=0,ori;
        ori=x;
        if(x<0)
            return false;
        while(x>0)
        {
            rev = (rev * 10) + (x % 10);
            x/=10;
        }
        if(rev==ori)
            return true;
        else
            return false;
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        ori = x
        rev = 0
        if(x < 0):
            return False
        while(x > 0):
        
            rev = (rev * 10) + (x % 10)
            x = x //  10
        
        if(rev == ori):
            return True
        return False